src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/PostgresDirectoryDAO.scala

```diff
@@ -343,11 +310,11 @@ class PostgresDirectoryDAO(protected val dbRef: DbReference,
                                        p: QuerySQLSyntaxProvider[SQLSyntaxSupport[PolicyRecord], PolicyRecord],
```

Comment:
we don't, but in the next PR I'm going to open that finishes off the pending directory dao tests we will need it, so I'm going to leave it for now since it would just get added right back in